src/layout/strategy/container.js

```diff
@@ -0,0 +1,29 @@
+export default function() {
+    var containerWidth = 0,
+        containerHeight = 0;
+
+    var container = function(point) {
+        return !(point.x < 0 || point.y < 0 ||
```

Comment:
it looks like this strategy returns booleans - I thought the others returned bounding boxes?